src/components/CioQuiz/quizApiReducer.ts

```diff
@@ -66,20 +68,26 @@ export default function apiReducer(
         quizFirstQuestion,
         quizResults: undefined,
         selectedOptionsWithAttributes: undefined,
+        matchedOptions: undefined,
       };
     }
     case QuizAPIActionTypes.SET_QUIZ_RESULTS: {
       const selectedOptionsWithAttributes =
         action.payload?.quizResults.quiz_selected_options
           .filter((option) => option.has_attribute)
           .map((option) => option.value) || [];
+      const matchedOptions =
+        action.payload?.quizResults.quiz_selected_options
+          .filter((option) => option.is_matched)
```

Comment:
Lint doesn't like this at the moment. I published a new version for `client-js` with the updated types. You can update the version to get rid of the lint issue